mmv1/templates/terraform/decoders/app_engine_domain_mapping.go.erb

```diff
@@ -0,0 +1,15 @@
+if v, ok := res["sslSettings"]; ok {
+  original := v.(map[string]interface{})
+
+  // sslManagementType does not get returned with the beta endpoint when set to "AUTOMATIC"
```

Comment:
```
              + ssl_settings {
                  + ssl_management_type = "MANUAL"
                }
```

OK- the beta API doesn't return it in either case. We'll want to update this comment and also set it to `MANUAL` when `pendingManagedCertificateId` is *not* present too.

Reply:
Thanks, updated as well.